tests/bot/exts/moderation/test_stream.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+
+
+from bot.constants import Roles
+from bot.exts.moderation.stream import Stream
+from tests.helpers import MockBot, MockMember, MockRole
+
+
+class StreamCommandTest(unittest.IsolatedAsyncioTestCase):
+
+    def setUp(self) -> None:
+        self.bot = MockBot()
+        self.cog = Stream(self.bot)
+
+    def test_checking_if_user_has_streaming_permission(self):
```

Comment:
This test doesn't call into any code in the cog, so I don't see the point in having it.

Reply:
@Harbys This still needs to be addressed. You either need to fix this test so it actually tests the cog, or decide that you don't want to work on tests (which is fine; we don't require tests) and remove this test file entirely.